mezzanine/__init__.py

```diff
@@ -1,2 +1,2 @@
 
-__version__ = "4.3.1"
+__version__ = "4.3.1.post1"
```

Comment:
Still PEP-440 happy.

Reply:
Cool
This is small enough that I guess spinning up a branch w/o this one change (and submitting just the below 3 lines as a PR) would be okay.